pkgs/development/python-modules/soundfile/aarch64-darwin.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/setup.py b/setup.py
+index b275081..517ef2a 100644
+--- a/setup.py
++++ b/setup.py
+@@ -16,6 +16,7 @@ MACOSX_VERSIONS = '.'.join([
+     'macosx_10_6_intel',
+     'macosx_10_9_intel',
+     'macosx_10_9_x86_64',
++    'macosx_11_0_arm64',
```

Comment:
this isn't nix-specific, do you mind creating an upstream patch?

best would to then use `fetchpatch` to pull from the PR

Reply:
Upstream are preparing a new 0.11 release https://github.com/bastibe/python-soundfile/issues/325, which reworked `setup.py` and fixed aarch64-darwin issue.

@bastibe Do you have any interest on this patch for 0.10 branch?

Reply:
Dear @azuwis, I will make note of the MACOSX_VERSIONS you used here, as you seem to employ a test suite that can ensure their correctness. I have struggled to find a working set of platform identifiers on macOS, so this is a big help. Thank you!

Reply:
> this isn't nix-specific, do you mind creating an upstream patch?
> 
> best would to then use `fetchpatch` to pull from the PR

Done, please review the new changes.